exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -91,7 +91,22 @@ drill.exec: {
     use.ip : false
   },
   optimizer: {
-    implementation: "org.apache.drill.exec.opt.IdentityOptimizer"
+    implementation: "org.apache.drill.exec.opt.IdentityOptimizer",
+    enhanced_defaults: {
```

Comment:
Any particular reason why you make this option as boot-time option, in stead of system option? Majority planner options, if not all, are able to reset either at system or session level. That will give the user option for better control.